### PR TITLE
Feature/apps 2454 add members to org

### PR DESCRIPTION
**Pull request by solomontang**
image

### CODE REVIEW

**Review by giuliacampana — APPROVED** [review]

File: src/core/redux/constants/index.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const IS_OWNER :'isOwner' = 'isOwner';
 export const USER_SEARCH_RESULTS :'userSearchResults' = 'userSearchResults';
 
 // TODO: does this belong here?
-export const INITIAL_SEARCH_RESULTS :Map = fromJS({ initial: true });
+export const INITIAL_SEARCH_RESULTS :Map = Map();
```
Comment (UnsungHero97) [inline src/core/redux/constants/index.js:43]: I added this as part of the old version of the app... there was a reason for it but I suspect we don't need to worry about it now.

File: src/containers/org/components/AddMemberModalBody.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import { ModalBody } from '../../../components';
+
+type Props = {
+  onChange :() => void;
+};
+
+const AddMemberModalBody = ({ onChange } :Props) => (
+  <ModalBody>
+    <Typography>Enter the username or email address of the person you wish to add to the organization</Typography>
```
Comment (UnsungHero97) [inline src/containers/org/components/AddMemberModalBody.js:16]: The "or email" bit requires a follow-up:

https://jira.openlattice.com/browse/APPS-2508

File: src/containers/org/components/SearchMemberBar.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+type Props = {
+  onChange :() => void;
+};
+
+const SearchMemberBar = ({ onChange } :Props) => {
+  const dispatch = useDispatch();
+  const [searchTerm, setSearchTerm] = useState('');
+  const searchRequestState = useRequestState([USERS, SEARCH_ALL_USERS]);
+  const userSearchResults = useSelector((store) => store.getIn([USERS, USER_SEARCH_RESULTS]));
```
Comment (UnsungHero97) [inline src/containers/org/components/SearchMemberBar.js:31]: As a follow-up to this pr, please create a selector for this.

File: src/containers/org/members/MembersSection.js [hunk 4]
```diff
@@ -143,10 +145,22 @@ const MembersSection = ({
 
   return (
     <MembersSectionGrid>
-      <MembersSectionHeader as="h4">{memberSectionHeader}</MembersSectionHeader>
+      <MembersSectionHeader as="h4">
```
Comment (UnsungHero97) [inline src/containers/org/members/MembersSection.js:148]: As a follow-up, let's switch to `Typography`.

**Review by UnsungHero97 — APPROVED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Review by giuliacampana — APPROVED** [review]

**Pull request merged**